webapp/components/navbar_dropdown.jsx

```diff
@@ -184,6 +187,20 @@ export default class NavbarDropdown extends React.Component {
                     </a>
                 </li>
             );
+        } else {
```

Comment:
This blocks are almost the same, shouldn't be best if the localize string is a variable and you only change that?